src/main/java/com/diablo3CharViewer/CharacterViewerManager.java

```diff
@@ -80,6 +88,18 @@ private boolean isHeroIDCorrect(String heroIdToCheck) {
         }
     }
 
+    private void showHeroesListForSpecificAccount(List<HeroDataModel> heroesOnProvidedAccount) {
+
+        for (HeroDataModel heroDataModel : heroesOnProvidedAccount) {
+            StringBuilder str = new StringBuilder();
```

Comment:
Could be moved to the HeroDataModel

Reply:
done;
Is that besause this method operates only on HeroDataModel fields?
I made this method static in HeroDataModel class.